misc/src/main/scala/com/danieltrinh/scalariform/gui/FormatterFrame.scala

```diff
@@ -294,6 +318,11 @@ class FormatterFrame extends JFrame with SpecificFormatter {
             preferences = preferences.setPreference(prefType.cast(preference), widget.asInstanceOf[JCheckBox].isSelected)
           case prefType @ IntegerPreference(min, max) ⇒
             preferences = preferences.setPreference(prefType.cast(preference), Integer.parseInt(widget.asInstanceOf[JSpinner].getValue.toString))
+          case prefType @ IntentPreference ⇒
+            val selected = widget.asInstanceOf[JPanel].getComponents.find(_.asInstanceOf[JRadioButton].isSelected).get.getName
```

Comment:
Sorry about that, I've pushed a new commit that fixes the Swing portions of this PR.